{"uuid": "c1ee0e0e-214b-4d6c-8a08-bb7d07d37c55", "children": ["87429203-656c-4a70-9ea2-9647fdb4c6e6"], "befores": [{"name": "setup_pages", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00EDFC83+61635]\n\tGetHandleVerifier [0x00EDFCC4+61700]\n\t(No symbol) [0x00D005D3]\n\t(No symbol) [0x00D4899E]\n\t(No symbol) [0x00D48D3B]\n\t(No symbol) [0x00D90E12]\n\t(No symbol) [0x00D6D2E4]\n\t(No symbol) [0x00D8E61B]\n\t(No symbol) [0x00D6D096]\n\t(No symbol) [0x00D3C840]\n\t(No symbol) [0x00D3D6A4]\n\tGetHandleVerifier [0x011645A3+2701795]\n\tGetHandleVerifier [0x0115FD26+2683238]\n\tGetHandleVerifier [0x0117AA6E+2793134]\n\tGetHandleVerifier [0x00EF6945+155013]\n\tGetHandleVerifier [0x00EFD02D+181357]\n\tGetHandleVerifier [0x00EE74D8+92440]\n\tGetHandleVerifier [0x00EE7680+92864]\n\tGetHandleVerifier [0x00ED2070+5296]\n\tBaseThreadInitThunk [0x76325D49+25]\n\tRtlInitializeExceptionChain [0x77C3D03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x77C3CFC1+561]\n\n", "trace": "  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\PAPI\\testcases\\test_new_regular_employee.py\", line 20, in setup_pages\n    self.employee_form_page.click_add_employee()\n  File \"C:\\PAPI\\pages\\new_regular_employee_page.py\", line 17, in click_add_employee\n    element = self.helpers.wait_for_element(LOC.ADD_EMPLOYEE_BUTTON, condition=\"clickable\")\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\PAPI\\utils\\helpers.py\", line 19, in wait_for_element\n    return self.wait.until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Sinergia\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1747819289670, "stop": 1747819310213}], "start": 1747819289670, "stop": 1747819310231}